tests/test_examples.py

```diff
@@ -7,7 +7,11 @@
 import pytest
 from cpmpy import *
 
-EXAMPLES = glob(join("..", "examples", "*.py")) + glob(join(".", "examples", "*.py")) + glob(join(".", "examples/advanced", "*.py"))
+EXAMPLES = glob(join("..", "examples", "*.py")) + \
+           glob(join(".", "examples", "*.py")) + \
+           glob(join("..", "examples/advanced", "*.py")) + \
+            glob(join("..", "examples/csplib", "*.py"))
+
```

Comment:
It would be better to put this into a separate pull request or merge directly with master.

Reply:
I agree, I'll put it in master already so I can test them for release